notes.md

```diff
@@ -233,7 +233,7 @@ Add `-vga vmware` to QEMU parameters in the booot script (e.g. boot-macOS.sh), i
 
 *Warning: The OpenCore distribution that comes with OSX-KVM already has AppleALC, do not mix VoodooHDA with AppleALC. You may want to consider HDA passthrough if it is practical or use HDMI audio instead*
 
-No support is provided for this. You are on your own. The sound output is known to be choppy and distorted.
+No support is provided for this. You are on your own. The sound output is known to be choppy and distorted. *(Though this can be mitigated using [spheenik's version of qemu](https://github.com/spheenik/qemu/tree/3.1.0-patched) but it has not been thoroughly tested)*
```

Comment:
There is a branch with QEMU 4.0 (https://github.com/spheenik/qemu/tree/4.0-glorious) but it should probably be omitted because we don't know how it'll behave